interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -156,9 +155,7 @@ def test_setCompositionDict(self):
         self.assertNear(X[0], 0.25)
         self.assertNear(X[3], 0.75)
 
-        # Change back to u'xx' syntax when support for Python 3.2 is dropped
-        #self.phase.Y = {u'H2':1.0, u'O2':3.0}
-        self.phase.Y = {'H2':1.0, 'O2':3.0}
+        self.phase.Y = {u'H2':1.0, u'O2':3.0}
```

Comment:
I don't think this is necessary; in Python 3, all strings are unicode strings. I think it would only be necessary for Python 2 support, and this appears to be testing the cython interface, and not `ctml_writer`. But this whole unicode/str/bytes business has me very confused...